classification/generate_sensor_data.py

```diff
@@ -27,27 +27,31 @@
 import os
 import random
 
-from settings import (SEQUENCE_LENGTH,
-                      DATA_DIR,
-                      DEFAULT_WHITE_NOISE_AMPLITUDE,
-                      WHITE_NOISE_AMPLITUDES,
-                      SIGNAL_AMPLITUDE,
-                      SIGNAL_MEAN,
-                      NUM_RECORDS,
-                      SIGNAL_PERIOD)
 
 
-
-def generateData(dataDir=None,
-                 signal_mean=SIGNAL_MEAN,
-                 signal_period=SIGNAL_PERIOD,
-                 number_of_points=NUM_RECORDS,
-                 signal_amplitude=SIGNAL_AMPLITUDE,
-                 noise_amplitude=DEFAULT_WHITE_NOISE_AMPLITUDE):
-  fileName = "white_noise_%s" % noise_amplitude
-
-  if not dataDir:
-    dataDir = DATA_DIR
+def generateData(dataDir,
```

Comment:
You use `open()` and `close()` for the file here. You should avoid that logic at all costs, and use the `with open()` context manager, **always**.